.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.DS_Store
+
+dst.txt
```

Comment:
Why do we need to ignore this file? If that is used in testing then we should save the file to tmp directory with https://www.npmjs.com/package/tmp

```
const tmp = require('tmp');
const file = tmp.fileSync();

console.log(file.name); // this is the file path

...


file.removeCallback(); // cleanup after the test
```